aspnetcore/publishing/iis.md

```diff
@@ -119,45 +123,51 @@ To include the *publish-iis* tool in your application, add entries to the *tools
 
-2. Set the application pool to **No Managed Code**. ASP.NET Core runs in a separate process and manages the runtime.
+4. Set the application pool to **No Managed Code**. ASP.NET Core runs in a separate process and manages the runtime.
 
-> [!NOTE]
-> If you change the default identity of the application pool from **ApplicationPoolIdentity**, verify the new identity has the required permissions to access the application's folder and database.
-
-Open the **Add Website** window.
+5. Open the **Add Website** window.
 
    ![Click Add Website from the Sites contextual menu.](iis/_static/add-website-context-menu-ws2016.png)
 
-Configure the website.
+6. Configure the website.
 
    ![Supply the Site name, physical path, and Host name in the Add Website step.](iis/_static/add-website-ws2016.png)
 
-In the **Application Pools** panel, open the **Edit Application Pool** window by right-clicking on the website's application pool and selecting **Basic Settings...** from the popup menu.
+7. In the **Application Pools** panel, open the **Edit Application Pool** window by right-clicking on the website's application pool and selecting **Basic Settings...** from the popup menu.
 
    ![Select Basic Settings from the contextual menu of the Application Pool.](iis/_static/apppools-basic-settings-ws2016.png)
 
-Set the **.NET CLR version** to **No Managed Code**.
+8. Set the **.NET CLR version** to **No Managed Code**.
 
    ![Set No Managed Code for the .NET CLR Version.](iis/_static/edit-apppool-ws2016.png)
+   
+> [!NOTE]
+> If you change the default identity of the application pool from **ApplicationPoolIdentity**, verify the new identity has the required permissions to access the application's folder and database.
 
-Browse the website.
+## Deploy the application
+Deploy the application to the folder you created on the target IIS server. Web Deploy is the recommended mechanism for deployment. Alternatives to Web Deploy are listed below.
 
-   ![The Microsoft Edge browser has loaded the IIS startup page.](iis/_static/browsewebsite.png)
+### Web Deploy with Visual Studio
+Create a publish profile in Visual Studio and click the **Publish** button to deploy your application. If your hosting provider has support for creating or supplying a publish profile, download their publish profile and import it using the Visual Studio Publish Web dialog.
+
+![Publish dialog page](iis/_static/pub-dialog.png)
+
+### Alternatives to Web Deploy
+If you don't wish to use Web Deploy or are not using Visual Studio, you may use any of several methods to move the application to the server, such as Xcopy, Robocopy, or PowerShell. Visual Studio users may use the [default Visual Studio web publish script](https://github.com/aspnet/vsweb-publish/blob/master/samples/default-publish.ps1).
```

Comment:
I wonder if it would be better to link to the [web publish script samples readme](https://github.com/aspnet/vsweb-publish/blob/master/samples/samples.md) rather than the "default web publish script"